docs/base-account/framework-integrations/privy/authentication.mdx

```diff
@@ -0,0 +1,225 @@
+---
+title: "Auth (Sign In With Base)"
+description: "Manage user authentication with Privy and Base Account"
+---
+
+import { GithubRepoCard } from "/snippets/GithubRepoCard.mdx";
+
+Learn how to handle authentication flows with Privy and Base Account, including both Privy-managed authentication and custom backend verification.
+
+## Overview
+
+Privy handles the initial authentication flow, managing user sessions and wallet connections. You can also implement additional authentication layers for enhanced security or custom requirements.
+
+The code snippets in this guide are based on the following example project:
+<GithubRepoCard
+  title="Base Account Privy Template"
+  githubUrl="https://github.com/base/base-account-privy"
+/>
+
+## Authentication Flow
+
+Privy manages the primary authentication before users enter your application:
+
+<div style={{ display: 'flex', justifyContent: 'center'}}>
+  <img src="/images/base-account/privy-base-auth.gif" alt="Privy Base Auth" style={{ width: '600px', height: 'auto' }} />
+</div>
+
+## Custom Authentication
+
+For additional security or custom authentication requirements, you can implement backend verification using Sign-In with Ethereum (SIWE)
+with the Base Account SDK.
+
+### Setup
+
+Follow the [Setup](/base-account/framework-integrations/privy/setup) guide to set up Privy with Base Account.
+
+### Frontend Component (Sign In With Base)
```

Comment:
maybe also include a step that explains where to use this `Authentication` component, like adding it to a page.tsx